### ISSUE TITLE
Mini Files borderless

### ISSUE THREAD

**Issue by peter-bread** [peter-bread/peter.nvim#52]
Just change normal fg to be same as bg and put it back on default borde style

**Comment by peter-bread** [comment]
```
      local MiniFilesBorder =
        vim.api.nvim_get_hl(0, { name = "MiniFilesBorder" })
      if MiniFilesBorder.link then
        local Link = vim.api.nvim_get_hl(0, { name = MiniFilesBorder.link })
        if Link.bg then
          vim.api.nvim_set_hl(
            0,
            "MiniFilesBorder",
            { fg = Link.bg, bg = Link.bg }
          )
        end
      end


```

**Comment by peter-bread** [comment]
Got it working now, but may break when changing colour schemes.

Globally store the links and read from them, might have to add autocmd for Color scheme change events

**Comment by peter-bread** [comment]
Closed in b2ed2869b44321029db0730dbcff6781efa4f83d.

Will need to be done on a per colorscheme basis.
